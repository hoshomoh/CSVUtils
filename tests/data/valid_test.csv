name,address,stars,contact,uri
Well Health Hotels,Inga N. P.O. Box 567,3,Kasper Zen,http://well.org
