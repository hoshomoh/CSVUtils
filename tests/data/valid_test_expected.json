[
    {
        "name": "Well Health Hotels",
        "address": "Inga N. P.O. Box 567",
        "stars": 3,
        "contact": "Kasper Zen",
        "uri": "http://well.org"
    }
]